deepmd/common.py

```diff
@@ -64,7 +65,11 @@ def gelu(x: tf.Tensor) -> tf.Tensor:
     Original paper
     https://arxiv.org/abs/1606.08415
     """
-    return op_module.gelu(x)
+    try:
+        gelu = tensorflow.nn.gelu
+    except AttributeError:
+        gelu = op_module.gelu
```

Comment:
`tf.nn.gelu` v.s. `op_module.gelu`, which is faster? 

If the latter is faster can we use it by default?

Reply:
The later is indeed fast, but sometimes an error will be reported during a long MD process, we need to be more careful.

Reply:
Maybe we could follow the implementations of tensorflow and do some more tests to figure out the reasons of this particular error.